gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -334,6 +334,11 @@
             <property name="message" value="printStackTrace is not generally allowed."/>
             <property name="ignoreComments" value="true"/>
         </module>
+        <module name="RegexpSinglelineJava">
+            <property name="format" value="System\.out\."/>
+            <property name="message" value="System.out usage is not generally allowed"/>
```

Comment:
The error message should explain why this is bad and what people should do instead.

Reply:
I added a reference to logging, have a look.